### PR TITLE
added documentation to current code

### PR DESCRIPTION
**Pull request by aidanthomas26**

### CODE REVIEW

File: Front-End/src/pages/Cashier/CashierNavbar.jsx
Comment (Daniel-Lam1702) [inline Front-End/src/pages/Cashier/CashierNavbar.jsx:1]: The CashierNavbar.jsx got replaced by the CashierOrderPage.jsx

File: Front-End/src/pages/Manager/InventoryManager.jsx
Comment (Daniel-Lam1702) [inline Front-End/src/pages/Manager/InventoryManager.jsx:1]: Please confirm with @Xach-A  the completeness of this file since we merged his changes.

**Review by Daniel-Lam1702 — CHANGES_REQUESTED** [review]
@aidanthomas26, looks good to me but please address my comments and request a second review.

File: Front-End/src/pages/Manager/InventoryManager.jsx
Comment (Xach-A) [reply Front-End/src/pages/Manager/InventoryManager.jsx:1]: I resolved the changes. This part is good now.

**Review by Daniel-Lam1702 — CHANGES_REQUESTED** [review]
Only need to fix the CashierNavbar.jsx.

**Review by Daniel-Lam1702 — APPROVED** [review]
Looks good to me.

**Pull request merged**